addon/scope.js

```diff
@@ -0,0 +1,60 @@
+class Scope {
```

Comment:
I was afraid that was the only way to do that.

I don't think there is the equivalent of that.

We could theoretically look through the ember test registry and declare those as methods?